src/sse/sse.js

```diff
@@ -45,6 +45,9 @@ This extension adds support for Server Sent Events to htmx.  See /www/extensions
           var internalData = api.getInternalData(parent)
           // Try to remove remove an EventSource when elements are removed
           if (internalData.sseEventSource) {
+            api.triggerEvent(parent, 'htmx:sseClose', {
+              type: 'nodeReplaced',
```

Comment:
Is there any reason you would not want to pass `source: internalData.sseEventSource` down the event, as you passed the event source reference in the 2 other event types below?

Reply:
I missed this one, thank you for noticing.